src/toolbars/SelectionBar.cpp

```diff
@@ -209,53 +208,73 @@ void SelectionBar::Populate()
    // Inner sizers have space on right only.
    // This choice makes for a nice border and internal spacing and places clear responsibility
    // on each sizer as to what spacings it creates.
-   wxFlexGridSizer *mainSizer = safenew wxFlexGridSizer(1, 1, 1);
+   wxFlexGridSizer *mainSizer = safenew wxFlexGridSizer(2, 1, 1);
    Add(mainSizer, 0, wxALIGN_CENTER_VERTICAL | wxLEFT, 5);
 
    // Top row (mostly labels)
    
    {
-      const wxString choices[4] = {
-         _("Start and End of Selection"),
-         _("Start and Length of Selection"),
-         _("Length and End of Selection"),
-         _("Length and Center of Selection"),
+      auto vSizer = std::make_unique<wxBoxSizer>(wxVERTICAL);
+      auto selectionText = AddTitle(XO("Selection"), vSizer.get());
+
+      auto setupBtn =
+         safenew AButton(this, wxID_ANY, wxDefaultPosition, { 20, 20 });
```

Comment:
When I build the commit that introduces this Button on my MacBook, the button fails to display and I don't know why.  There is just a blank space on the toolbar below the title.

Reply:
I also observed at that commit that if you drag the toobar off the dock, it does not give you a resizer grip at the corner as other toolbars do.

Reply:
I notice now, most toolbars did not have the resizer grip.  The two new ones do not.  The few that do have it are the meters and play-at-speed, for which it makes sense, and also the Time toolbar.

The Time toolbar adapts the numeric control to a bigger font or smaller.

Maybe there is a question for design @LWinterberg @DilsonsPickles @Tantacrul whether that feature would also be desirable for the other toolbars that have this numeric control too -- the selection spectral selection toolbars (that last is hidden by default).

Or maybe we just ignore that question until after rebasing to Qt.